### PR TITLE
Cocktailsection

### PR DESCRIPTION
**Pull request by sorrento974**

### CODE REVIEW

File: client/src/components/CocktailCard.jsx [hunk 1]
```diff
@@ -25,8 +24,8 @@ CocktailCard.propTypes = {
   drink: PropTypes.shape({
     strDrink: PropTypes.string.isRequired,
     strDrinkThumb: PropTypes.string.isRequired,
-    strInstructions: PropTypes.string.isRequired,
-    idDrink: PropTypes.number.isRequired,
+    strInstructions: PropTypes.string,
```
Comment (JulienChevreauPro) [inline client/src/components/CocktailCard.jsx]: useless

File: client/src/main.jsx [hunk 2]
```diff
@@ -3,14 +3,10 @@ import ReactDOM from "react-dom/client";
 import { createBrowserRouter, RouterProvider } from "react-router-dom";
 
 import App from "./App";
-
-// page components
```
Comment (JulienChevreauPro) [inline client/src/main.jsx:7]: Remove comments please

File: client/src/main.jsx [hunk 3]
```diff
@@ -3,14 +3,10 @@ import ReactDOM from "react-dom/client";
...
-
 import Home from "./pages/HomePage";
 import CocktailPage from "./pages/CocktailPage";
 import CategoryPage from "./pages/CategoryPage";
-
-// router creation
```
Comment (JadeJolivet) [inline client/src/main.jsx:13]: remove comments

File: client/src/main.jsx [hunk 4]
```diff
@@ -50,6 +46,20 @@ function allCocktails() {
     .then((data) => data.drinks);
 }
 
+function getPopularCocktails() {
+  return fetch(
+    "https://www.thecocktaildb.com/api/json/v1/1/filter.php?c=Cocktail"
```
Comment (JadeJolivet) [inline client/src/main.jsx]: endpoint to change to ordinaryDrink

File: client/src/pages/ContactPage.jsx [hunk 5]
```diff
@@ -0,0 +1,104 @@
+import "../styles/ContactPage.css";
+import { useState } from "react";
+
+function ContactForm() {
+  const [formData, setFormData] = useState({
+    lastname: "",
+    firstname: "",
+    mail: "",
```
Comment (JulienChevreauPro) [inline client/src/pages/ContactPage.jsx]: replace by email please

File: client/src/main.jsx [hunk 6]
```diff
@@ -94,6 +114,10 @@ const router = createBrowserRouter([
         element: <CategoryPage />,
         loader: () => mocktails(),
```
Comment (JadeJolivet) [inline client/src/main.jsx]: change "mocktails" to "getMocktails"

File: client/src/styles/CategoryPage.css [hunk 7]
```diff
@@ -19,4 +20,11 @@
     padding: 6.5rem 3rem 2rem 3rem;
     grid-template-columns: repeat(3, 1fr);
   }
+  .categoryPage > :last-child {
+    grid-column: 2 / -3;
```
Comment (JulienChevreauPro) [inline client/src/styles/CategoryPage.css]: -3?

File: client/src/styles/ContactPage.css [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+  min-height: 50px;
+  padding: 8px;
+  border-radius: 5px;
+  border: 1px solid #ccc;
+  box-shadow: 0 1px 1px rgba(0, 0, 0, 0.1);
+  box-sizing: border-box;
+  font-family: var(--secondaryFont);
+  font-size: 1.1rem;
+}
+#message {
```
Comment (JadeJolivet) [inline client/src/styles/ContactPage.css]: useless

File: client/src/styles/CocktailCard.css [hunk 9]
```diff
@@ -1,6 +1,7 @@
 .cocktailCard {
   display: flex;
   flex-direction: column;
+  place-items: center;
   align-items: center;
```
Comment (JulienChevreauPro) [inline client/src/styles/CocktailCard.css]: useless

**Review by JadeJolivet — APPROVED** [review]

**Review by JulienChevreauPro — APPROVED** [review]

**Review by JadeJolivet — APPROVED** [review]

**Review by JadeJolivet — APPROVED** [review]

**Review by JadeJolivet — APPROVED** [review]

**Review by JulienChevreauPro — APPROVED** [review]

**Review by JadeJolivet — APPROVED** [review]

File: client/src/components/Footer.jsx [hunk 10]
```diff
@@ -4,7 +4,13 @@ import "../styles/Footer.css";
 function Footer() {
   return (
     <footer>
-      <img src="./src/assets/images/logo.svg" alt="Logo Cocktail Club" />
+      <a href="/">
```
Comment (Yoann-Fortin) [inline client/src/components/Footer.jsx]: Failed on line goal :(

**Review by Yoann-Fortin — CHANGES_REQUESTED** [review]

**Review by JadeJolivet — APPROVED** [review]

**Review by JadeJolivet — APPROVED** [review]

**Review by JulienChevreauPro — APPROVED** [review]

**Review by Elea641 — APPROVED** [review]

**Pull request merged**